component/block_cache/block_cache.go

```diff
@@ -261,8 +261,8 @@ func (bc *BlockCache) Configure(_ bool) error {
 		return fmt.Errorf("config error in %s [memory limit too low for configured prefetch]", bc.Name())
 	}
 
-	log.Info("BlockCache::Configure : block size %v, mem size %v, worker %v, prefeth %v, disk path %v, max size %vMB, disk timeout %v",
-		bc.blockSize, bc.memSize, bc.workers, bc.prefetch, bc.tmpPath, bc.diskSize, bc.diskTimeout)
+	log.Info("BlockCache::Configure : block size %v, mem size %v, worker %v, prefetch %v, disk path %v, max size %vMB, disk timeout %v, prefetch-on-open %t, blockPool %v, diskPolicy %v, maxDiskUsageHit %v, noPrefetch %v, threadPool %v",
+		bc.blockSize, bc.memSize, bc.workers, bc.prefetch, bc.tmpPath, bc.diskSize, bc.diskTimeout, bc.prefetchOnOpen, bc.blockPool, bc.diskPolicy, bc.maxDiskUsageHit, bc.noPrefetch, bc.threadPool)
```

Comment:
threadpool is an object and not a value, you need to print the number of threads configured instead. same with block pool as well.

Reply:
The number of threads is already printed by `bc.workers`